### ISSUE TITLE
Your units conversion service seems to be down... 

### ISSUE THREAD

**Issue by lendresen87** [LLNL/units#205]
Hi, your units conversion service mentioned in the docs seems to be down: 
https://units.readthedocs.io/en/latest/_static/convert.html

Do you know when the service will be up again?

**Comment by lendresen87** [comment]
Yeah, but we are looking into a more permanent solution, as the usage got a bit higher than expected.

**Comment by phlptp** [comment]
The webserver is pretty easy to run locally.  Right now it is just running a docker image from docker hub on an AWS instance.  You can run that docker locally or just compile the webserver and run that on whatever you like.   Curious what you are doing with it.  

And it looked like the docker server just stopped for some unknown reason, so I just restarted it.

**Comment by lendresen87** [comment]
Cool, so in theory, we could set up something in our AWS and use that instead? 
We have an app that generates some configuration files that we configure our hardware with. We are doing some readings on pressure and temperature and such, and needed a way to convert to the desired units. 
We've been calling your REST API for that, but the number of conversions were higher than expected, so I am a bit worried we are calling your service more than it was built for. We have a meeting today to set up something running on our side, so that should fix it. Ps, great library that you have built :)

**Comment by lendresen87** [comment]
So I am currently trying to build a docker image using your Docker file in the webserver folder, but it seems to be failing to find the units_webserver in the build folder.
```
#13 0.387 cp: cannot stat '../build/webserver/units_webserver': No such file or directory
#13 ERROR: executor failed running [/bin/sh -c cp ../build/webserver/units_webserver .]: exit code: 1
```

Do you have any theories to what could be wrong? 
Also (can't hurt to ask), do you have an official docker image that is available to the public?

**Comment by lendresen87** [comment]
So, I had to change this line in your Docker file order to make it build the image: 
`RUN cp ../build/webserver/units_webserver . && cp ../units/webserver/*.html . && cp ../units/webserver/*.sh .`
to 
`RUN cp ../build/bin/units_webserver . && cp ../units/webserver/*.html . && cp ../units/webserver/*.sh .`

I also updated the version label to 0.5.0.

**Comment by phlptp** [comment]
hmm,  haven't looked at the dockerhub in a while.  Guess it isn't updating anymore, looks like the webhooks changed

Anyway
https://hub.docker.com/repository/docker/phlptp/units/general
is supposed to be where some docker images get generated and are publicly available but looks like it isn't updating.  I will see if I can get that working again and update the dockerfile.

**Comment by lendresen87** [comment]
Thank you, that would be super :)

**Comment by phlptp** [comment]
The docker hub images are updated and working again

Here is a script I run on my EC2 instance
```sh
#!/usr/bin/bash

docker logs units_webserver &> /home/ec2-user/weblog/weblog$(date "+%s").log

docker stop units_webserver

docker container rm units_webserver

docker pull phlptp/units:webserver

docker run --name units_webserver -p80:80 -d phlptp/units:webserver

``` 
I think I run this once a day to automatically get the latest images and regenerate the server if it had stopped

**Comment by i-ky** [comment]
> regenerate the server if it had stopped

You can use Docker [restart policies](https://docs.docker.com/engine/reference/commandline/run/#restart-policies---restart) in combination with health check command (either [in Dockerfile](https://docs.docker.com/engine/reference/builder/#healthcheck) or as [`docker run` options](https://docs.docker.com/engine/reference/commandline/run/#options)).
